metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -173,6 +173,22 @@ public void testMapConstant() {
   }
 
   @Test
+  public void testArithmetic() {
+    {
+      String query = "1 + 2";
+      Assert.assertEquals(3, ((Number)run(query, ImmutableMap.of("casey", "casey"))).doubleValue(), 1e-3);
+    }
+    {
+      String query = "1.2 + 2";
+      Assert.assertEquals(3.2, ((Number)run(query, ImmutableMap.of("casey", "casey"))).doubleValue(), 1e-3);
+    }
+    {
+      String query = "1.2e-3 + 2";
+      Assert.assertEquals(1.2e-3 + 2, ((Number)run(query, ImmutableMap.of("casey", "casey"))).doubleValue(), 1e-3);
+    }
```

Comment:
The `ImmutableMap.of("casey","casey")`(besides promoting your global brand ;) ) do nothing for these tests, right?  It'd be more clear, if we removed those.  Everything else looks good.  I'm a +1 if we can take care of this small issue.